pkg/kudoctl/cmd/install/install.go

```diff
@@ -181,59 +207,95 @@ func verifySingleFramework(name, previous string, gc *github.Client, k2c *kudo.C
 			if helpers.AskForConfirmation() {
 				// If Instance is a dependency we need to make sure installSingleInstanceToCluster is aware of it.
 				// By having the previous string set we can make this distinction.
-				err := installSingleInstanceToCluster(name, previous, *content.Path, gc, k2c)
+				err := installSingleInstanceToCluster(name, previous, bundlePath, kc)
 				if err != nil {
 					return errors.Wrap(err, "installing single Instance")
 				}
 			}
 		} else {
-			err := installSingleInstanceToCluster(name, previous, *content.Path, gc, k2c)
+			err := installSingleInstanceToCluster(name, previous, bundlePath, kc)
 			if err != nil {
 				return errors.Wrap(err, "installing single Instance")
 			}
 		}
 
 	}
-
 	return nil
 }
 
 // Todo: needs testing
 // installSingleFrameworkToCluster installs a given Framework to the cluster
-func installSingleFrameworkToCluster(name, path string, gc *github.Client, k2c *kudo.Client) error {
-	frameworkYaml, err := gc.GetFrameworkYaml(name, path)
+func installSingleFrameworkToCluster(name, path string, kc *kudo.Client) error {
+	frameworkPath := path + "/" + name + "-framework.yaml"
+	frameworkYamlFile, err := os.Open(frameworkPath)
+	if err != nil {
+		return errors.Wrap(err, "failed opening framework file")
+	}
+
+	frameworkByteValue, err := ioutil.ReadAll(frameworkYamlFile)
 	if err != nil {
-		return errors.Wrapf(err, "getting %s-framework.yaml", name)
+		return errors.Wrap(err, "failed reading framework file")
 	}
-	_, err = k2c.InstallFrameworkObjToCluster(frameworkYaml)
+
+	var f v1alpha1.Framework
+	err = yaml.Unmarshal(frameworkByteValue, &f)
+	if err != nil {
+		return errors.Wrapf(err, "unmarshalling %s-framework.yaml content", name)
+	}
+
+	_, err = kc.InstallFrameworkObjToCluster(&f)
 	if err != nil {
 		return errors.Wrapf(err, "installing %s-framework.yaml", name)
 	}
-	fmt.Printf("framework.%s/%s created\n", frameworkYaml.APIVersion, frameworkYaml.Name)
+	fmt.Printf("framework.%s/%s created\n", f.APIVersion, f.Name)
 	return nil
 }
 
 // Todo: needs testing
 // installSingleFrameworkVersionToCluster installs a given FrameworkVersion to the cluster
-func installSingleFrameworkVersionToCluster(name, path string, gc *github.Client, k2c *kudo.Client) error {
-	frameworkVersionYaml, err := gc.GetFrameworkVersionYaml(name, path)
```

Comment:
I think GetFrameworkVersionYaml can be removed now

Reply:
I think I removed it 🤔 where do you see it?

Reply:
I've talked to @alenkacz and this has been removed when we removed the GithubClient.